package.json

```diff
@@ -129,7 +129,7 @@
     }
   },
   "engines": {
-    "node": ">= 10.*"
+    "node": ">= 14.*"
```

Comment:
While working on #1674, running `ember s` or `ember t` resulted in an error (this happened even in the `main` branch at the time of writing):

```sh
building... [Babel: ember-test-helpers > applyPatches]node:internal/validators:164
      throw new ERR_INVALID_ARG_TYPE(name, 'Object', value);
      ^

TypeError [ERR_INVALID_ARG_TYPE]: The "options" argument must be of type object. Received an instance of Array
    at ChildProcess.target.send (node:internal/child_process:733:7)
    at Array.forEach (<anonymous>)
    at dispatchQueuedRequests (/ember-intl/node_modules/@ember/test-helpers/node_modules/workerpool/lib/WorkerHandler.js:174:21)
    at ChildProcess.<anonymous> (/ember-intl/node_modules/@ember/test-helpers/node_modules/workerpool/lib/WorkerHandler.js:129:7)
    at ChildProcess.emit (node:events:527:28)
    at ChildProcess.emit (node:domain:475:12)
    at emit (node:internal/child_process:938:14)
    at processTicksAndRejections (node:internal/process/task_queues:84:21) {
  code: 'ERR_INVALID_ARG_TYPE'
}
```

According to https://github.com/vercel/vercel/pull/7782, we'll want to pin Node to `14`.

(While working on this pull request, I didn't reliably reproduce and remove the error. In CI, running `ember test` with Node 16.15.1 also resulted in the error above. Locally, with 16.15.0, I was able to run `ember test`, however.)

Reply:
Possibly related: https://github.com/ember-cli/ember-cli/issues/9404#issuecomment-792335093